Source/Atomic/Resource/Image.cpp

```diff
@@ -246,7 +247,9 @@ Image::Image(Context* context) :
     components_(0),
     cubemap_(false),
     array_(false),
-    sRGB_(false)
+    sRGB_(false),
+    pot_(false),
+    hasAlphaChannel_(false)
 {

```

Comment:
These new cached member variables which are invalidated if IsPowerOfTwo or components > 3 change, can these be simple getters which return whether image is pot or has alpha channel querying IsPowerOfTwo(width_) && IsPowerOfTwo(height_) and components > 3